.github/workflows/ci.yml

```diff
@@ -47,14 +47,19 @@ jobs:
     - name: List installed Python packages
       run: python -m pip list
 
-    - name: Test with pytest
+    - name: Test with pytest and coverage
       run: |
-        pytest --ignore tests/benchmarks/ --ignore tests/contrib --ignore tests/test_notebooks.py
+        coverage run --module pytest --ignore tests/contrib --ignore tests/benchmarks --ignore tests/test_notebooks.py
```

Comment:
Need to now manually run under `coverage` and the `--module` is just how you tell it to run something like `pytest`, and not a local runner script.